### DISCUSSION TITLE
Ich komme nicht ins Start Menü

### DISCUSSION THREAD

**Discussion by KHackmann** [jomjol/AI-on-the-edge-device#2377] [Q&A]
Wenn ich per EDGE oder Crome auf die IP meines ESP32 gehe, kommt diese Fehlermeldung. 

.__(.)< ( oh oh! This page does not exist! )
    \___)

               You could try your [luck](http://192.168.178.112/index.html) here!

Wo muss ich ansetzen um den Fehler zu finden? 
Danke für Eure Hilfe.

**Comment by caco3** [comment]
Ich vermute, Du hast die SD-Karte nicht richtig aufgesetzt. -> https://jomjol.github.io/AI-on-the-edge-device-docs/Installation/#3-sd-card

Was sagt das Log (`http://<IP>/log`)?
